running locally/docker-prestissimo/docker-compose.yml

```diff
@@ -0,0 +1,119 @@
+version: '1'
+services:
+  minio:
+    hostname: minio-server
+    image: 'minio/minio:latest'
+    container_name: minio-server
+    command: server --console-address ":9001" /minio
+    ports:
+      - '9000:9000'
+      - '9001:9001'
+    environment:
+      AWS_ACCESS_KEY_ID: minioadmin
+      AWS_SECRET_ACCESS_KEY: minioadmin
+    volumes:
+      - '~/minio/data:/minio'
```

Comment:
home folder?

Reply:
This is where the local data is actually stored for the MinIO container.